models/models.go

```diff
@@ -20,6 +20,12 @@ const (
 	StepRunStatusCodeSkippedWithRunIf = 4
 	// StepRunStatusCodePreparationFailed ...
 	StepRunStatusCodePreparationFailed = 5
+	// StepRunStatusAborted ...
```

Comment:
nit: Now that we don't have that linter that forces comments on public fields, can we slowly remove these meaningless comments from those places we change?